osu.Game.Rulesets.Taiko/UI/DrumTouchInputArea.cs

```diff
@@ -173,8 +232,23 @@ private partial class QuarterCircle : CompositeDrawable, IKeyBindingHandler<Taik
 
             public override bool Contains(Vector2 screenSpacePos) => circle.Contains(screenSpacePos);
 
-            public QuarterCircle(TaikoAction handledAction, Color4 colour)
+            public QuarterCircle(TaikoAction handledAction)
             {
+                Color4 colour = ((Func<Color4>)(() =>
+                {
+                    #pragma warning disable format
```

Comment:
why are you disabling and enabling the formatting instead of just formatting it  in such a way as to avoid the complaint, i get its more compact this way but still

Reply:
It's a readability thing
This:
![image](https://user-images.githubusercontent.com/48618519/211546208-89569969-a7e2-4ff3-a791-a730701b2be4.png)
Reads way nicer than this:
![image](https://user-images.githubusercontent.com/48618519/211546348-b7f7b644-229b-403f-a160-f5e9f75624fa.png)

Reply:
my point was moreso just putting it on the next line, but eitherway i guess the core devs can decide whether this is better or worse